pings.yaml

```diff
@@ -9,6 +9,18 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
+study-enrollment:
+  description: |
+    This ping is sent when the user has consented to the study.
+  include_client_id: false
+  send_if_empty: true
```

Comment:
Out of curiosity: Why is this flag `true` here? From what I see in the code the `uid` _always_ has a value.